docs/node/getting-started.md

```diff
@@ -39,60 +39,83 @@ $ npm install @bufbuild/buf @bufbuild/protoc-gen-es @bufbuild/protobuf @connectr
 
 ## Define a service
 
-First, we need to add a Protobuf file that includes our service definition. For
-this tutorial, we are going to construct a unary endpoint for a service that is
-a stripped-down implementation of [ELIZA](https://en.wikipedia.org/wiki/ELIZA),
-the famous natural language processing program.
+We have already defined the `ElizaService` and have
+it [hosted on the Buf Schema Registry][eliza-proto] to use in this example.
+Export the Protobuf schema to use in our project.
 
 ```bash
-$ mkdir -p proto && touch proto/eliza.proto
+$ npx buf export buf.build/connectrpc/eliza -o proto
 ```
 
-Open up the above file and add the following service definition:
+Our new `proto` directory should look like this:
+```
+.
+└ proto
+  └── connectrpc
+      └── eliza
+          └── v1
+              └── eliza.proto
```

Comment:
The next page "Implementing services" builds on this page, and imports from `"./gen/eliza_connect"`. If we choose to use the directory tree (I'm not opposed to it), these imports have to be updated as well. There may be more pages that need to be updated.